docs/content/dagster-cloud/managing-deployments/dagster-cloud-yaml.mdx

```diff
@@ -0,0 +1,296 @@
+
+The `dagster_cloud.yaml` file should be placed in the root of your Dagster project. Below is an example of a file structure from the [Dagster cloud ETL quickstart](https://github.com/dagster-io/dagster/tree/master/examples/quickstart_etl).
+
+```shell
+quickstart_etl
+├── README.md
+├── quickstart_etl
+│   ├── __init__.py
+│   ├── assets
+├── quickstart_etl_tests
+├── dagster_cloud.yaml
+├── pyproject.toml
+├── setup.cfg
+└── setup.py
+```
+
+If your repository contains multiple Dagster projects in subdirectories - otherwise known as a monorepository - add the `dagster_cloud.yaml` file to the root of each project subdirectory.
+
+## Settings schema
+
+### Code locations
+
+The only **mandatory** section in the `dagster_cloud.yaml` file is the `locations` key. This top-level key specifies a list of code locations that should be deployed to Dagster Cloud.
+
+At a minimum, each item in `locations` must include the following: 
+- A `location_name` key that specifies the name of the code location
+- A `code_source` key containing either a `module_name` or `file_name` parameter that specifies where to find the definitions in the code location
+
+
+Refer to the following table for the full list of parameters a location can have.
+
+<TabGroup>
+<TabItem name="Single code location">
+
+```yaml
+# dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+```
+
+</TabItem>
+<TabItem name="Multiple code locations">
+
+```yaml 
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+  - location_name: machine_learning
+    code_source:
+      python_file: ml/ml_model.py
+```
+</TabItem>
+</TabGroup>
+
+### Working directory
+
+Use the `working_directory` setting to load Dagster code from a different directory than the root of your code repository. This setting allows you to specify the directory you want to load your code from.
+
+
+Consider the following project:
+
+```shell
+quickstart_etl
+├── README.md
+├── project_directory
+│   ├── quickstart_etl
+│     ├── __init__.py
+│     ├── assets
+│   ├── quickstart_etl_tests
+├── dagster_cloud.yaml
+├── pyproject.toml
+├── setup.cfg
+└── setup.py
+```
+
+To load from `/project_directory`, the `dagster_cloud.yaml` code location would look like this:
+
+```yaml
+# dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+    working_directory: ./project_directory
+```
+
+### Build directory
+
+The build registry is applicable to hybrid deployments only. Setting a build directory is useful if your `setup.py` or `requirements.txt` is in a subdirectory instead of the project root. This is common if you have multiple Python packages within a single Dagster project.
+
+In the example below, the docker image for the code location is in the root directory and the registry and image defined. 
+
+Example:
```

Comment:
Let's take this copy out - you stated above that there's an example, no need to state it again